pyfolio/tears.py

```diff
@@ -76,8 +77,8 @@ def create_returns_tear_sheet(returns, live_start_date=None,
         benchmark_rets = utils.get_symbol_rets('SPY')
         # If the strategy's history is longer than the benchmark's, limit
         # strategy
-        benchmark_start = benchmark_rets.index[0].astimezone(pytz.UTC)
-        returns_start = returns.index[0].astimezone(pytz.UTC)
+        benchmark_start = get_utc_date(benchmark_rets.index[0])
```

Comment:
I wonder if we should just pass in the right thing from research. Either the returns or the benchmark must be tz-naive.

For general input checking I think it'd be great to use https://github.com/TomAugspurger/engarde.